app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -155,10 +155,10 @@ function toggleCodeOfConduct() {
   }
   var codeOfConduct = user.checked_code_of_conduct
   var commentCount = user.number_of_comments
-  var checkboxWrapper = document.getElementById('toggle-code-of-conduct-checkbox')
+  var checkboxWrapper = document.getElementById('toggle-code-of-conduct-checkbox');
   if (checkboxWrapper && !codeOfConduct && commentCount < 1) {
     checkboxWrapper.innerHTML = '<input type="checkbox" name="checked_code_of_conduct" class="checkbox" required/>\
-                                  <label for="checked_code_of_conduct"/>I\'ve read the <a href="https://dev.to/code-of-conduct">code of conduct</a></label>'
+                                  <label for="checked_code_of_conduct"/>I\'ve read the <a href="<%= ApplicationConfig["APP_PROTOCOL"] %><%= ApplicationConfig["APP_DOMAIN"] %>/code-of-conduct">code of conduct</a></label>'
```

Comment:
I'm not sure if maybe theres a way that I'm not aware of to pull in the app helper in this file, instead of using <%= ApplicationConfig["APP_PROTOCOL"] %><%= ApplicationConfig["APP_DOMAIN"] %>. This was the most effective way.

Reply:
Nice one thanks @fdoxyz !